Cargo.toml

```diff
@@ -19,25 +19,27 @@ include = [
 [dependencies]
 anyhow = "1.0"
 async-trait = "0.1"
-aws-config = "0.56"
-aws-sdk-athena = "0.29"
-aws-sdk-s3 = "0.29"
-aws-sdk-sqs = "0.29"
-aws-smithy-http = "0.56"
-aws-types = "0.56"
+aws-sdk-athena = "1.10.0"
+aws-config = { version = "1.1.2", features = ["behavior-version-latest"] }
+aws-sdk-s3 = "1.12.0"
+aws-sdk-sqs = "1.10.0"
+aws-smithy-runtime-api = "1.1.2"
+aws-smithy-async = "1.1.2"
+aws-types = "1.1.2"
 aws_lambda_events = { version = "0.13", default-features=false, features = ["sqs"]}
-lambda_runtime = "0.9"
+lambda_runtime = "0.9.0"
 bytesize = "1.3"
 clap = { version = "4.4", features = ["derive", "env"] }
 derivative = "2.2"
 futures = "0.3"
-http = "0.2"
+http = "1.0"
 serde = "1.0"
 serde_json = "1.0"
 tokio = { version = "1.35", features=["macros"] }
 tracing = "0.1"
 tracing-subscriber = { version = "0.3", features=["json", "env-filter"] }
 url = { version = "2.5", features = ["serde"] }
+bytes = "1.5.0"
```

Comment:
**nitpick:** To help with readability these dependencies are maintained in alphabetical order. Could we move this new dependency up?

(This is another case where a linter would be an ideal solution to keep things tidy (in the eye of the beholder)).